app/src/renderer/main.js

```diff
@@ -6,7 +6,7 @@ import Vuelidate from 'vuelidate'
 import shrinkStacktrace from '../helpers/shrink-stacktrace.js'
 import axios from 'axios'
 import Raven from 'raven-js'
-import {remote} from 'electron'
+import { remote } from 'electron'
```

Comment:
i am in favour of these stylistic changes, but we should consider (agreeing on and) using a linter so no one changes this back / future styles are the same.

Reply:
👍 
vs code was messed up for a moment and reformatted stuff. Wrote in the chat about agreeing on one rule.